http/rest/service.go

```diff
@@ -0,0 +1,67 @@
+package rest
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"raccoon/collection"
+	"raccoon/config"
+	"raccoon/http/rest/websocket"
+	"raccoon/http/rest/websocket/connection"
+	"raccoon/metrics"
+	"time"
+
+	"github.com/gorilla/mux"
+)
+
+type Service struct {
+	Buffer chan *collection.CollectRequest
```

Comment:
Can we make the collector as a struct field instead of the channel, currently we are initializing collector the collector twice whereas only one can be used?

Reply:
Done